javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,20 @@
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2022;
+let dogYearFuture = yearFuture;
+const difference1 = dogYearFuture - dogYearOfBirth;
+const dogYear = 7;
+const humanYear = 1;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true) {
+  const result2 = difference1 * dogYear;
+  console.log(
+    "Your dog will be " + result2 + " human years old in " + yearFuture
+  );
+} else if (shouldShowResultInDogYears === false) {
```

Comment:
Again, AFAIK it should be covered in week2, but small advice:
shouldShowResultInDogYears might be only `true` or `false`. 
So it would be simpler to do
```
if (shouldShowResultInDocYears) {
   console.log('Your dog will be ... dog years ...')
} else {
   console.log('Your dog will be ... human years ...')
}
```